### ISSUE TITLE
Esercitazione 25 mar 2024 - Q2

### ISSUE THREAD

**Issue by luckignolo32** [CS-Swap/Calcolo-Integrale#27]
![25mar2](https://github.com/CS-Swap/Calcolo-Integrale/assets/147436677/b07236de-32c3-43cf-be0e-6da64ac8b373)

**Comment by notedo** [comment]
Falso, per convergere ad f la funzione dev'essere analitica e per essere analitica non basta che la funzione sia derivabile infinite volte
